### PR TITLE
Bump flutter, dart and hover

### PR DESCRIPTION
**Pull request by ericdallo**
flutter: 1.20.2 -> 1.22.2
dart: 2.9.1 -> 2.10.0
~hover: 0.43 -> 0.44.0~ (latest version not working on nix for some reason yet)

### CODE REVIEW

File: pkgs/flutter/default.nix [hunk 1]
```diff
@@ -1,14 +1,11 @@
-{ pkgs }:
+{ flutterPackages }:
 
-let
-  mkFlutter = opts: pkgs.callPackage (import ./flutter.nix opts) {};
-in
-mkFlutter rec {
+flutterPackages.mkFlutter rec {
```
Comment (thiagokokada) [inline pkgs/flutter/default.nix:3]: I think this will only work on unstable right now :sadpanda:.

Comment (ericdallo) [reply pkgs/flutter/default.nix:3]: I thought this was added on 20.09, isn't ?

Comment (thiagokokada) [reply pkgs/flutter/default.nix:3]: Is 20.09 already released :thinking: ?

Comment (thiagokokada) [reply pkgs/flutter/default.nix:3]: OMG IT IS!!!!!!!!!!!!

Comment (ericdallo) [reply pkgs/flutter/default.nix:3]: HAHAHHAHAHA

Comment (thiagokokada) [reply pkgs/flutter/default.nix:3]: Can we merge this PR next week? I should upgrade my system on this weekend, so I can test your changes too.

**Review by thiagokokada — APPROVED** [review]

**Pull request merged**